test/verify/machineslib.py

```diff
@@ -196,7 +224,7 @@ def tearDown(self):
     def startLibvirt(self):
         m = self.machine
         # Ensure everything has started correctly
-        m.execute("systemctl start libvirtd.service")
+        m.execute("systemctl restart libvirtd.service")
```

Comment:
Is that still necessary? The test cleanup should always stop it now.

Reply:
Right, that should not be necessary, it slipped through.